design-library/source/_patterns/01-atoms/13-search/search.json

```diff
@@ -0,0 +1,6 @@
+{
+  "search_id": "search",
+  "search_name": "Blog search",
+  "search_placeholder": "Search",
+  "search_icon": "/images/svg/search.svg"
```

Comment:
We may want to hardcode this because it won't change

Reply:
I m changing this